project.clj

```diff
@@ -105,7 +106,7 @@
                        :build-type "foss"
                        :main-namespace "puppetlabs.puppetdb.main"
                        :create-varlib true}
-                :config-dir "ext/config"
+                :config-dir "ext/config/foss"
```

Comment:
No, `ext/config` actually wasn't even being used (that directory doesn't exist).

I mainly added `foss` subdirectory here to demonstrate a further means of differentiation between `foss` and `pe` if necessary. I don't see any problem with both the `foss` and the `pe` build pointing to the same directory, especially since they aren't being used--just let me know what your preference is and I will update the PR.